src/edu/ucsb/nceas/metacat/properties/BackupPropertiesDelegate.java

```diff
@@ -0,0 +1,284 @@
+package edu.ucsb.nceas.metacat.properties;
+
+import edu.ucsb.nceas.metacat.service.ServiceService;
+import edu.ucsb.nceas.metacat.shared.MetacatUtilException;
+import edu.ucsb.nceas.metacat.shared.ServiceException;
+import edu.ucsb.nceas.metacat.util.SystemUtil;
+import edu.ucsb.nceas.utilities.GeneralPropertyException;
+import edu.ucsb.nceas.utilities.MetaDataProperty;
+import edu.ucsb.nceas.utilities.PropertiesMetaData;
+import edu.ucsb.nceas.utilities.PropertyNotFoundException;
+import edu.ucsb.nceas.utilities.SortedProperties;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import javax.xml.transform.TransformerException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.Set;
+import java.util.Vector;
+
+public class BackupPropertiesDelegate {
+    static boolean bypassAlreadyChecked = false;
+    // Full Path to the DIRECTORY containing the configuration BACKUP files (such as
+    // metacat.properties.backup etc.). Example: /var/metacat/.metacat
+    static Path backupDirPath;
+    private static SortedProperties mainBackupProperties = null;
+
+    private static BackupPropertiesDelegate instance = null;
+    String mainBackupFilePath = null;
+    private static final Log logMetacat = LogFactory.getLog(PropertiesWrapper.class);
+
+
+    private BackupPropertiesDelegate()
```

Comment:
Javadoc -- last time I will raise it, but fix throughout.

Reply:
PropertyService, AuthPropertiesDelegate & BackupPropertiesDelegate addressed in commit 7a41ed9274d9c0e63125b03add225c38537dae91